shacl/index.html

```diff
@@ -461,7 +461,7 @@
 	foaf:mbox &lt;mailto:bob@example.org&gt; ;
 	foaf:mbox &lt;mailto:rs@example.org&gt; .</pre>
 				<p>
-					The SHACL validation would validate <code>inst:Issue1</code> and <code>inst:Issue3</code> against <code>ex:IssueShape</code>
+					The SHACL validation would validate data graphs <code>inst:Issue1</code> and <code>inst:Issue3</code> against shapes graph<code>ex:IssueShape</code>
```

Comment:
On 4/24/16 6:41 AM, Dimitris Kontokostas wrote:
> In shacl/index.html
> <https://github.com/w3c/data-shapes/pull/18#discussion_r60844762>:
>
>> @@ -461,7 +461,7 @@
>>  	foaf:mbox &lt;mailto:bob@example.org&gt; ;
>>  	foaf:mbox &lt;mailto:rs@example.org&gt; .</pre>
>>  				<p>
>> -					The SHACL validation would validate <code>inst:Issue1</code> and <code>inst:Issue3</code> against <code>ex:IssueShape</code>
>> +					The SHACL validation would validate data graphs <code>inst:Issue1</code> and <code>inst:Issue3</code> against shapes graph<code>ex:IssueShape</code>
>
> I would change it to:
> "The SHACL validation would validate |inst:Issue1| and |inst:Issue3| in
> the data graph against shapes graph|ex:IssueShape| in the shapes graph"
> or leave it as is.
> putting data graph and shapes graph in front of the resources is like
> the resources are the actual graphs while these resources only exist as
> subjects in triples in the respecting graphs

Changing the order is ok, and to me it means the same thing in either 
order. - kc

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/data-shapes/pull/18/files/52fded8308a039314d783a385110ed702c951dcb#r60844762>
>

-- 
Karen Coyle
kcoyle@kcoyle.net http://kcoyle.net
m: 1-510-435-8234
skype: kcoylenet/+1-510-984-3600